### PR TITLE
Added GNOME

### PR DESCRIPTION
**Pull request by slashtechno**
> this should have been made a native pi-apps script (its as simple as `install_packages gnome gnome-core gnome-software gnome-shell`)  

_Originally posted by @theofficialgman in https://github.com/Botspot/pi-apps/issues/685#issuecomment-1135345893_
            

I edited the install and uninstall scripts to use `install_packages` and `purge_packages` respectively.

### CODE REVIEW

File: apps/GNOME/install [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+install_packages gnome gnome-core gnome-software gnome-shell ||
```
Comment (theofficialgman) [inline apps/GNOME/install:3]: missing exit 1

File: apps/GNOME/uninstall [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+purge_packages gnome gnome-core gnome-software gnome-shell || error 1
```
Comment (theofficialgman) [inline apps/GNOME/uninstall:3]: purge_packages is all you call. it does not take any input

also error 1 is incorrect. that will write `1` to the terminal and exit

File: apps/GNOME/description [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Installs the GNOME Desktop
+Install GNOME desktop and removes Raspberry Pi OS desktop
```
Comment (theofficialgman) [inline apps/GNOME/description:2]: incorrect. this does not remove the piOS desktop

File: apps/GNOME/uninstall (hunk 2 above)
Comment (slashtechno) [reply apps/GNOME/uninstall:3]: Ah, okay, that makes sense. If I'm understanding correctly,  `error` outputs an error message and exits with exit code `1` whilst `exit` exits with the specified error code - correct?

Comment (theofficialgman) [reply apps/GNOME/uninstall]: yes

File: apps/GNOME/uninstall [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+purge_packages gnome gnome-core gnome-software gnome-shell || error "Could not remove GNOME and its software."
```
Comment (theofficialgman) [inline apps/GNOME/uninstall:3]: still wrong
purge_packages does not take any input
you just call purge_packages || exit 1
that is it

File: apps/GNOME/install [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+install_packages gnome gnome-core gnome-software gnome-shell || error "Could not install GNOME and its software."
```
Comment (theofficialgman) [inline apps/GNOME/install:3]: install_packages already prints to the terminal what it failed to install
you don't need to write any more output, just use exit 1

File: apps/GNOME/credits [hunk 6]
```diff
@@ -0,0 +1 @@
+TerraGitHuB (inital script) - YT channel: https://www.youtube.com/channel/UCjCHOhXCgQRBHCY8IeBZTzQ
```
Comment (theofficialgman) [inline apps/GNOME/credits:1]: no credit shoud be given to anyone but the gnome developers
this isn't special or unique

Comment (slashtechno) [reply apps/GNOME/credits:1]: I was wondering what the credits should be. I didn't add/remove parties in the credits as I wasn't sure.

**Comment by slashtechno** [comment]
Tried this on ARM64 Pi OS and it didn't install.

**Comment by slashtechno** [comment]
Seems some package was broken.

File: apps/GNOME/uninstall [hunk 7]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+purge_packages gnome gnome-core gnome-software gnome-shell || exit 1
```
Comment (theofficialgman) [inline apps/GNOME/uninstall:3]: still wrong... not going to say why as I have already said it multiple times now

**Comment by theofficialgman** [comment]
> Seems some package was broken.

can confirm that debian and raspbian based systems seem broken here
works on all ubuntu versions. see github actions test build: https://github.com/Botspot/pi-apps/actions/runs/4236698055

**Comment by slashtechno** [comment]
> > Seems some package was broken.
> 
> can confirm that debian and raspbian based systems seem broken here
> works on all ubuntu versions. see github actions test build: https://github.com/Botspot/pi-apps/actions/runs/4236698055

Ah, okay.
Is there any solution to this?

**Comment by theofficialgman** [comment]
> > > Seems some package was broken.
> > 
> > 
> > can confirm that debian and raspbian based systems seem broken here
> > works on all ubuntu versions. see github actions test build: https://github.com/Botspot/pi-apps/actions/runs/4236698055
> 
> Ah, okay. Is there any solution to this?

ask about it in raspberry pi forums
for now this will be closed

**Pull request closed without merging**